.github/actions/setup-cargo-semver-checks/action.yml

```
@@ -0,0 +1,7 @@
+name: "Setup cargo semver-checks"
+description: "Installs the cargo semver-checks tool"
+runs:
+  using: "composite"
+  steps:
+    - run: wget -q -O- https://github.com/obi1kenobi/cargo-semver-checks/releases/download/v0.18.2/cargo-semver-checks-x86_64-unknown-linux-gnu.tar.gz | tar -xz -C ~/.cargo/bin
```

```suggestion
    - run: wget -q -O- https://github.com/obi1kenobi/cargo-semver-checks/releases/download/v0.18.3/cargo-semver-checks-x86_64-unknown-linux-gnu.tar.gz | tar -xz -C ~/.cargo/bin
```
v0.18.3 is released and all prior 0.18.x are yanked due to a CLI parsing bug that could lead to false-negatives

---

Thank you!